.teamcity/src/main/kotlin/common/JvmCategory.kt

```diff
@@ -21,7 +21,7 @@ enum class JvmCategory(
     override val version: JvmVersion
 ) : Jvm {
     MIN_VERSION(JvmVendor.oracle, JvmVersion.java8),
-    MAX_VERSION(JvmVendor.adoptiumopenjdk, JvmVersion.java17),
+    MAX_VERSION(JvmVendor.adoptiumopenjdk, JvmVersion.java18),
```

Comment:
We now test JDK 8/11/17 in different stages. With this change, we'll lost coverage for 17, which is the latest LTS - I don't think this is expected. My suggestion is that we test with Java 18 in QuickFeedbackLinuxOnly, and leave other jdk17 builds unchanged. To achieve this goal, we need to change these lines to hardcoded `jdk17` instead of `JvmCategory.MAX_VERSION`: https://github.com/gradle/gradle/blob/c00251daa0aeaada2287bcfa770d73949e62444b/.teamcity/src/main/kotlin/model/CIBuildModel.kt#L80

Reply:
@blindpirate WDYT about adding something like QF at the Ready For Release stage that builds with the latest JDK release?

The coverage would be...

- QF:
   - quickTest Linux + latest LTS
   - quickTest Windows + earliest LTS
- PRF:
   - platformTest Linux + earliest LTS
   - platformTest Windows + latest LTS
- RFN:
   - Quick cross version testing + earliest LTS 
- RFR:
   - Full cross version testing and other scenarios
   - platformTest macOS + earliest LTS
   - **quickTest Linux + latest EA** (new)
   - **quickTest Windows + latest EA** (new)

Reply:
This is probably the best because I expect less people will use latest non-LTS.

Reply:
I went with the original plan of only running JDK 18 under Check Quick Linux. I have an idea for a better coverage setup, but I need to get familiar with CI before I can implement it, and I don't want it to clog up merging this PR. I will make a new PR for those changes.